modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py

```diff
@@ -493,17 +515,21 @@ def deploy_ray_func(func, *args):  # pragma: no cover
         Function to be executed on an axis partition.
     *args : iterable
         Additional arguments that need to passed in ``func``.
+    **kwargs : dict
+        Additional keyword arguments to be passed in `func`.
 
     Returns
     -------
-    list
+    list : Union[tuple, list]
         The result of the function ``func`` and metadata for it.
 
     Notes
     -----
     Ray functions are not detected by codecov (thus pragma: no cover).
     """
-    result = func(*args)
+    if "args" in kwargs:
+        kwargs["args"] = deserialize(kwargs["args"])
```

Comment:
What about deserialization of other arguments which are not included in "args"? Is there such a case?

Reply:
There doesn't appear to be this case. In a previous commit, `func`, `args`, `kwargs`, and `kwargs["args"]` were all deserialized. I am not opposed to adding them back, because there's little overhead to `deserialize` if it's not necessary, and if it is necessary, it would break without `deserialize`. Would adding them back (as a defensive) be okay @YarShev ?

Reply:
I would suggest thinking of a more explicit passing arguments to remote functions (not through *args and *kwargs) to clearly understand what we need to deserialize and what not. Let's leave that for #4473 or even for another one.